src/Gremlin.Net.CosmosDb/Serialization/AutoConnector.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Gremlin.Net.CosmosDb.Structure;
+
+namespace Gremlin.Net.CosmosDb.Serialization
+{
+    internal class AutoConnector : IVertexConnector, IEdgeConnector
```

Comment:
Can you do me a favor and add some comments throughout this class and methods just to help future devs?